pytest_mh/_private/plugin.py

```diff
@@ -562,6 +567,22 @@ def _clone_function(self, name: str, f: pytest.Function) -> pytest.Function:
             originalname=f.originalname,
         )
 
+    def _preferred_topology(self, item: pytest.Item):
+        if self.mh_ignore_preferred_topology:
+            self.logger.info(" --mh-ignore-preferred-topology set: Ignoring preferred topology marker!")
+            return
+
+        if len(self.mh_topology) > 0:
+            self.logger.info(" --mh-topology set: Ignoring preferred topology marker!")
+            return
```

Comment:
This is logged for each test. Once, during pytest initialization, is more than enough. At https://github.com/next-actions/pytest-mh/blob/master/pytest_mh/_private/plugin.py#L202 to make it visible.